.golangci.yml

```diff
@@ -50,6 +50,8 @@ linters:
     # - lll
 
 linters-settings:
+  goimports:
+    local-prefixes: github.com/ava-labs/avalanchego
```

Comment:
q: what is this for?

Reply:
https://golangci-lint.run/usage/linters/#goimports